pkg/monitor/monitor/monitor_manager.go

```diff
@@ -598,3 +588,108 @@ func (m *MonitorManager) patchPVClaimRef(pv *corev1.PersistentVolume, patchPvcNa
 	}
 	return nil
 }
+
+type FakeMonitorManager struct {
+	MonitorManager
+}
+
+func NewFakeMonitorManager() *FakeMonitorManager {
+	return &FakeMonitorManager{}
+}
+
+func (m *FakeMonitorManager) patchTidbClusterStatus(tc *v1alpha1.TidbCluster, monitor *v1alpha1.TidbMonitor) error {
+	return nil
+}
+
+func (m *FakeMonitorManager) SyncMonitor(monitor *v1alpha1.TidbMonitor) error {
```

Comment:
This is not a good way.
Please refer to https://github.com/pingcap/tidb-operator/blob/master/pkg/manager/meta/reclaim_policy_manager_test.go#L233 to create fake MonitorManager.